client/src/www/app/app.ts

```diff
@@ -444,28 +445,29 @@ export class App {
   }
 
   private requestAddServer(event: CustomEvent) {
-    try {
-      this.serverRepo.add(event.detail.accessKey);
-    } catch (err) {
-      this.changeToDefaultPage();
-      this.showLocalizedError(err);
-    } finally {
-      this.rootEl.$.addServerView.open = false;
-    }
+    this.serverRepo
+      .add(event.detail.accessKey)
+      .catch(err => {
```

Comment:
Any reasons why not changing this function to `async` function?

Reply:
It's never awaited. It's called from a sync event handler